1. pam_systemd doesn't exist on void, use pam_elogind instead
2. pam_turnstile allows turnstile to integrate with lightdm

See also: https://github.com/chimera-linux/cports/blob/master/main/gdm/patches/pam.patch

--- a/data/pam/lightdm
+++ b/data/pam/lightdm
@@ -17,4 +17,6 @@
 
 # Setup session
 session   required pam_unix.so
+- session   optional pam_turnstile.so
+- session   optional pam_elogind.so
 session   optional pam_systemd.so
--- a/data/pam/lightdm-autologin
+++ b/data/pam/lightdm-autologin
@@ -19,4 +19,6 @@
 
 # Setup session
 session   required pam_unix.so
+- session   optional pam_turnstile.so
+- session   optional pam_elogind.so
 session   optional pam_systemd.so
--- a/data/pam/lightdm-greeter
+++ b/data/pam/lightdm-greeter
@@ -14,4 +14,6 @@
 
 # Setup session
 session   required pam_unix.so
+- session   optional pam_turnstile.so
+- session   optional pam_elogind.so
 session   optional pam_systemd.so
